test/api/bundle.test.ts

```diff
@@ -91,40 +82,23 @@ test('pack', () => {
     allowedLicenses: ['Apache-2.0', 'MIT'],
   });
 
+  // we need to first fix all violations
+  // before we can pack
+  bundle.validate({ fix: true });
+
   bundle.pack();
 
   const tarball = path.join(pkg.dir, `${pkg.name}-${pkg.version}.tgz`);
-  expect(fs.existsSync(tarball)).toBeTruthy();
 
-});
+  const workdir = fs.mkdtempSync(os.tmpdir());
+  shell(`npm install ${tarball}`, { cwd: workdir });
 
-test('validate and fix', () => {
```

Comment:
Removed since validation with fix is now called (and asserted on) from the packing test